### ISSUE TITLE
"Pause" winget after it finished

### ISSUE THREAD

**Issue by NCC1701M** [microsoft/winget-cli#1599]
### Description of the new feature / enhancement

If you run winget by using "PowerToys Run" the command line window where winget is running in is closing immediately after winget has finished.
If the winget command, e.g. `winget install` has completed successfully it might be ok that there is no final information of the status. But if the command failed or did not succeed for some reason, you have no idea why the command failed. The command line window closes so fast, you can't see any message.

For example:
The command `winget install draw.io` results in the output
```
Multiple packages found with matching input criteria. Please refine the input.
Name             ID           Source
-------------------------------------
draw.io Diagrams 9MVVSZK43QQW msstore
draw.io          JGraph.Draw  winget
```
But you can't see this information because the command line window closes directly after the output was written.

### Proposed technical implementation details

You should at least pause the program if the command could not be executed successfully, so that the user can read the "error" message.

One benefit of the "PowerToys" Run window is, that you can run every command as administrator only by pressing Shift+Ctrl+Enter. This is much easier and faster then the "old" way.

**Comment by denelon** [comment]
I'd hate to introduce a pause on "all" users for operations. We might consider some kind of a pause switch. Optionally, we can see if @crutkas has some ideas.

**Comment by sredna** [comment]
>  some ideas.

Try to detect if there are no other processes in the console

```
DWORD pids[2];
DWORD count = GetConsoleProcessList(pids, 2);
bool doPause = count == 1;
```

**Comment by jedieaston** [comment]
Maybe it would be better for PowerToys Run to have an option to append `; pause` after every command, so you can see the output of what you ran.

**Comment by denelon** [comment]
I think we can look to add a "--pause" switch, and the corollary setting to users can add a pause after each command is executed.

**Comment by Trenly** [comment]
@denelon - Resolved with #2236
